rl_algorithms/fd/ddpg_learner.py

```diff
@@ -0,0 +1,128 @@
+import argparse
+from typing import Tuple, Union
+
+import torch
+from torch.nn.utils import clip_grad_norm_
+
+from rl_algorithms.common.abstract.learner import TensorTuple
+import rl_algorithms.common.helper_functions as common_utils
+from rl_algorithms.ddpg.learner import DDPGLearner
+from rl_algorithms.utils.config import ConfigDict
+
+
+class DDPGfDLearner(DDPGLearner):
+    """Learner for DDPGfD Agent
+
+    Attributes:
+        args (argparse.Namespace): arguments including hyperparameters and training settings
+        hyper_params (ConfigDict): hyper-parameters
+        optim_cfg (ConfigDict): config of optimizer
+        log_cfg (ConfigDict): configuration for saving log and checkpoint
```

Comment:
should remove because it is written in abstract class.

Reply:
We agreed to decide what (not) to write on descriptions later.